source/edge-server/manage-edge-servers.txt

```diff
@@ -182,17 +182,54 @@ registration key.
 
             .. include:: /includes/api-authenticate-instructions.rst
 
-         .. step:: Call the Create Edge Server Endpoint
+         .. step:: Call the Create Edge Server Instance Endpoint
 
-            Call the <CREATE EDGE SERVER> endpoint to add an Edge Server 
-            instance to your service. You must provide the the App's internal 
-            ``appId`` hex string:
+            Call the 
+            :admin-api-endpoint:`Admin API Create an Edge Server instance <operation/adminCreateEdgeServerInstance>` 
```

Comment:
Note for reviewer: these `:admin-api-endpoint:` directives point to the live version of the Admin API spec: https://github.com/mongodb/snooty-parser/blob/03154c50f954010a279499a2bbf5c8bfb7b5bb58/snooty/rstspec.toml#L1535

Since the feature branch contains the API spec updates for Edge Server, all of these links will fail until we publish the feature branch. We can ignore broken link build errors related to these directives as this is an expected failure.